src/libraries/System.Reflection.Metadata/src/System/Reflection/PortableExecutable/ManagedTextSection.cs

```diff
@@ -187,7 +187,7 @@ private int ComputeOffsetToMetadata()
 
         public int ComputeSizeOfTextSection()
         {
-            Debug.Assert(MappedFieldDataSize % MappedFieldDataAlignment == 0);
+            // Debug.Assert(MappedFieldDataSize % MappedFieldDataAlignment == 0);
```

Comment:
I'm a bit leary of this. It doesn't look right to uncomment this. Either it is wrong and can be updated or removed. I see this is in Draft so can I assume it is a point in time change?

Reply:
> I see this is in Draft so can I assume it is a point in time change

Yep, ss discussed offline this will be reverted